Cargo.toml

```diff
@@ -60,6 +60,7 @@ members = [
   "test",                               # Integration tests.
   "test-rpc-functions",                 # RPC functions specifically for tests.
   "test-utils",                         # Various utilities for tests.
+  "trezor-common",                      # Code used by Trezor firmware repository.
```

Comment:
Would `wallet` folder be a better place for it?

Reply:
moved